homework/src/index.js

```diff
@@ -30,18 +33,84 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { html: err.message, class: 'alert-error' });
-      } else {
-        createAndAppend('pre', root, { html: JSON.stringify(data, null, 2) });
+  const HYF_REPOS_URL = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
+  const root = document.getElementById('root');
+  createAndAppend("img", root, { src: "./hyf.png", id: "imgLogo" });
+  const divSelect = createAndAppend("header", root);
+  createAndAppend("p", divSelect, { html: "HYF Repositories : ", id: "pContributors" });
+  const repoSelect = createAndAppend("select", divSelect);
+  const divFlex = createAndAppend("section", root);
+  const divInfo = createAndAppend("article", divFlex, { id: "divInfo" });
+  const divCont = createAndAppend("article", divFlex, { id: "divCont" });
+
+  function main(HYF_REPOS_URL) {
+
+    fetchJSON(HYF_REPOS_URL)
+      .catch(error => {
+        const divError = createAndAppend("container", root, { id: "divError" });
+        divError.innerHTML = "";
+        divError.innerHTML = error;
+        return;
```

Comment:
This return is useless. It just return from the arrow function passed to catch(). It would return anyway on the ending curly brace.

Reply:
Corrected